README.md

```diff
@@ -57,7 +56,7 @@ pod install
 ```
 
 ## Sample applications
-* **Bdk Wallet Demo App:** The [Bdk Wallet Demo App](https://github.com/LtbLightning/bdk-flutter-demo-app)
+* **BDK Flutter Demo App:** The [BDK Flutter Demo App](https://github.com/LtbLightning/bdk-flutter-demo-app)
```

Comment:
Demo App link is wrong. It should be [https://github.com/LtbLightning/bdk-flutter-app](https://github.com/LtbLightning/bdk-flutter-app)

Reply:
Thanks, will address it now.